src/HttpClient/Response.php

```diff
@@ -4,20 +4,100 @@
 namespace EoneoPay\Externals\HttpClient;
 
 use EoneoPay\Externals\HttpClient\Interfaces\ResponseInterface;
-use EoneoPay\Utils\Arr;
 use EoneoPay\Utils\Collection;
+use Psr\Http\Message\ResponseInterface as PsrResponseInterface;
+use Zend\Diactoros\MessageTrait;
 
 final class Response extends Collection implements ResponseInterface
 {
+    use MessageTrait;
+
     /**
-     * @var string
+     * Map of standard HTTP status code/reason phrases
+     *
+     * This array was copy and pasted shamelessly from the Zend Diactoros
+     * implementation because it wasnt part of the trait.
+     *
+     * @var string[]
      */
-    private $content;
+    private static $phrases = [
```

Comment:
I'm really surprised we can't already reference this from elsewhere in the codebase.

Reply:
Yeah, its unfortunate but theres not much to be done if we want to conform to the Psr interfaces

Reply:
Ah right, fair enough.